acceptance-tests/support/steps/clients-steps.ts

```diff
@@ -0,0 +1,65 @@
+import {Then} from "@cucumber/cucumber";
+import {TestContext} from "../test-setup";
+import {strict as assert} from "assert";
+
+const fields = {
+    "redirect uris": "redirectUris",
+    "post logout redirect uris": "postLogoutRedirectUris",
+    "user attributes": "userAttributes",
+    "first redirect uri": "redirectUri1"
+};
+
+Then("they should see the value for the Client ID {string}", async function (this: TestContext, text) {
+    const bodyText = await this.page.$eval("#client-id-value", element => element.textContent);
+    assert.equal(bodyText, text);
+});
+
+Then("they should only see the first 24 characters of the public key for their service", async function (this: TestContext) {
+    const keyContainerContent = await this.page.$eval("#publicKeyShort", element => element.textContent);
+    const elementVisibility = await this.page.$eval("#publicKeyShort", element => element.checkVisibility());
+    assert.equal(keyContainerContent!.length, 28);
```

Comment:
```suggestion
    assert.equal(keyContainerContent.length, 28);
```
This should get rid of the linting issue (illegal non-null assertion)